corehq/apps/users/dbaccessors/all_commcare_users.py

```diff
@@ -39,7 +39,7 @@ def _get_es_query(domain, user_filters):
     return query
 
 
-def get_commcare_users_by_filters(domain, user_filters, count_only=False):
+def get_commcare_users_by_filters(domain, user_filters, accessible_location_ids=None, count_only=False):
```

Comment:
Know if it's straightforward to include `accessible_location_ids` as another one of `user_filters`?  Sounds like it's in-line with that concept.

Reply:
I thought that too but wanted to keep what user entered and what HQ added separate, so that there is a clear distinction.